### PR TITLE
feat: make block error codes an environment variable

### PR DESCRIPTION
**Pull request by gabel0287**
*Issue #, if available:*
#75 

*Description of changes:*
Updated the cloudformation template and python log parser to create a configurable environment variable for the status codes to block

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: source/log-parser/log-parser.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 # Constants
 #======================================================================================================================
 API_CALL_NUM_RETRIES = 3
-BLOCK_ERROR_CODES = ['400','403','404','405'] # error codes to parse logs for
+BLOCK_ERROR_CODES = environ['BLOCK_ERROR_CODES']# error codes to parse logs for
```
Comment (arunsivasankaran) [inline source/log-parser/log-parser.py]: what happens if this is not set?

Comment (arunsivasankaran) [reply source/log-parser/log-parser.py]: may be set a default value?

Comment (gabel0287) [reply source/log-parser/log-parser.py]: @arunsivasankaran default value is set in the terraform template

**Comment by hnishar** [comment]
Thanks for opening a pull-request ticket. We will review the requests to be implemented in our next release.

**Comment by hvital** [comment]
Thanks for the contribution. We incorporated the idea and extracted all common customizations to a JSON config file. 

More info [here](https://docs.aws.amazon.com/solutions/latest/aws-waf-security-automations/appendix-c.html)

**Pull request closed without merging**